homeassistant/components/deconz/climate.py

```diff
@@ -115,13 +124,21 @@ def __init__(self, device, gateway):
         """Set up thermostat device."""
         super().__init__(device, gateway)
 
-        self._hvac_mode_to_deconz = dict(HVAC_MODE_TO_DECONZ)
+        # if settings climate SPZB0001 auto is heat
+        if self._device.modelid == "SPZB0001":
+            self._hvac_mode_to_deconz = dict(HVAC_MODE_TO_DECONZ_SPZB0001)
+        else:
+            self._hvac_mode_to_deconz = dict(HVAC_MODE_TO_DECONZ)
+
         if "mode" not in device.raw["config"]:
             self._hvac_mode_to_deconz = {
                 HVAC_MODE_HEAT: True,
                 HVAC_MODE_OFF: False,
             }
-        elif "coolsetpoint" not in device.raw["config"]:
+        elif (
+            "coolsetpoint" not in device.raw["config"]
+            and self._device.modelid != "SPZB0001"
+        ):
```

Comment:
```suggestion
        elif "coolsetpoint" not in device.raw["config"]:
```

Reply:
We need this because the Spirit has only two modes and no coolsetpoint so that would throw an error.

Reply:
That is managed by the if statement on line 128